### ISSUE TITLE
Recommendations to fine-tune to single class?

### ISSUE THREAD

**Issue by fipeop** [wasserth/TotalSegmentator#159]
Thanks for the great work! Do you have any recommendations to fine-tune the model if I have thousands of high quality annotations for a single class (e.g. pancreas)? 

Does formatting the training segmentations with "0" no class (any non-pancreas area) and "10" pancreas  would work? Or is it better to pass the scan data through the pretained model, then correct the pancreas predictions with the ground truth labels, and use these for fine-tuning?

**Comment by wasserth** [comment]
I never fine-tune my models. If I have new/more data I either train a new model from scratch or add the data to my entire training dataset and train the model again. Training takes roughly 1-2days on 1 GPU. So not too bad.
Therefore I can not really give any advice on fine-tuning.

**Comment by jumbojing** [comment]
I also have this issue again. For example, I only want to segment the spine instead of the vertebral as the terminology you provided...
